include/ableton/link/Controller.hpp

```diff
@@ -119,7 +119,7 @@ using StartStopStateCallback = std::function<void(bool)>;
 template <typename PeerCountCallback,
   typename TempoCallback,
   typename StartStopStateCallback,
-  typename Clock,
+  typename ClockT,
```

Comment:
We should not rename `Clock` to stay consistent with the rest of the code base.

Reply:
But then the code becomes confusing because there is a Clock declaration somewhere for the default clock isn't it?

Reply:
I'd like to stick to `Clock`. From the `Controller`'s perspective nothing really changed regarding the default clock. There is only the type alias in `Config.hpp` and then the clock is injected into the `Controller` and the rest of Link. The difference is that the `Controller` type alias is now declared in `Link.hpp` instead of `Config.hpp`.

Reply:
Done.